configs/efficientnet/retinanet_effb3_fpn_crop896_8x4_1x_coco.py

```diff
@@ -0,0 +1,87 @@
+_base_ = [
+    '../_base_/models/retinanet_r50_fpn.py',
+    '../_base_/datasets/coco_detection.py', '../_base_/default_runtime.py'
+]
+
+cudnn_benchmark = True
```

Comment:
is this setting necessary?

Reply:
I directly copy the setting of retinanet_r50_fpn_crop640_50e_coco.py in nas_fpn.

Reply:
OK, we can keep that for now. In the next step we should verify the effectiveness of this option and leave a comment to explain if this key is necessary, otherwise we can delete them.